apps/mobile/src/features/Onboarding/components/OnboardingCarousel/OnboardingCarousel.test.tsx

```diff
@@ -1,8 +1,18 @@
 import React from 'react'
 import { OnboardingCarousel } from './OnboardingCarousel'
-import { Text, View } from 'tamagui'
+import { Text } from 'tamagui'
 import { render } from '@/src/tests/test-utils'
 
+jest.mock('@notifee/react-native', () => ({
+  AndroidImportance: {
+    NONE: 0,
+    MIN: 1,
+    LOW: 2,
+    DEFAULT: 3,
+    HIGH: 4,
+  },
+}))
+
```

Comment:
I guess that we should move this to a global mock as you have it also in the Onboarding.container.test.tsx?